x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel.test.tsx

```diff
@@ -7,16 +7,17 @@
 
 import React from 'react';
 import { act } from 'react-dom/test-utils';
```

Comment:
We can import `act` from '@testing-library/react` to consolidate these import statements ([doc](https://testing-library.com/docs/react-testing-library/api/#act))